akka-stream/src/main/scala/akka/stream/javadsl/Tcp.scala

```diff
@@ -147,6 +147,8 @@ class Tcp(system: ExtendedActorSystem) extends akka.actor.Extension {
   /**
    * Creates an [[Tcp.OutgoingConnection]] instance representing a prospective TCP client connection to the given endpoint.
    *
+   * Note that framing (element boundary) may not be retained through this Flow.
+   *
```

Comment:
I think this is more misleading than not having this comment to start with, if there is no framing there is no framing.

I'm on the fence having anything at all, because this is something you need to understand to do network programming and if you don't already, I'm not sure in general a short comment will help.

If there should be something I think it should be more along the lines of:
`Note that the ByteString chunk boundaries is not retained across the network, to achieve application level chunks you have to introduce explicit framing in your streams, for example using the [[Framing]] stages.`